### PR TITLE
430: /summary command does not support multi-line summaries

### PR DESCRIPTION
**Pull request by edvbld**
Hi all,

please review this patch that makes the `/summary` pull request command accept multi-line summaries. As described on the [Skara wiki](https://wiki.openjdk.java.net/display/SKARA/Pull+Request+Commands#PullRequestCommands-/summary) the `/summary` command should this, since the commit message format allow for this.

Testing:
- [x] Added a new unit test
- [x] `make test` passes on Linux x64

Thanks,
Erik

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

### Issue
 * [SKARA-430](https://bugs.openjdk.java.net/browse/SKARA-430): /summary command does not support multi-line summaries

### Reviewers
 * Robin Westberg ([rwestberg](@rwestberg) - **Reviewer**) ⚠️ Review applies to c0ca352e75a5e66694fa5843ca204e60a9d8650d

### Download
`$ git fetch https://git.openjdk.java.net/skara pull/678/head:pull/678`
`$ git checkout pull/678`

### CODE REVIEW

File: bots/pr/src/main/java/org/openjdk/skara/bots/pr/SummaryCommand.java [hunk 1]
```diff
@@ -27,32 +27,45 @@
...
+                reply.println(Summary.setSummaryMarker(summary));
             }
         } else {
-            if (currentSummary.isPresent()) {
-                reply.println("Updating existing summary to `" + args.strip() + "`");
-            } else {
-                reply.println("Setting summary to `" + args.strip() + "`");
-            }
-            reply.println(Summary.setSummaryMarker(args.strip()));
+            // A single-line summary
```
Comment (rwestberg) [inline bots/pr/src/main/java/org/openjdk/skara/bots/pr/SummaryCommand.java:64]: Check if lines.size > 1 here? Perhaps that should be an error / warning?

Comment (edvbld) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/SummaryCommand.java:64]: Hm, yeah, good comment. Let me fix that!

Comment (edvbld) [reply bots/pr/src/main/java/org/openjdk/skara/bots/pr/SummaryCommand.java:77]: Fixed in 202a7d3

**Pull request closed without merging**